src/formBuilder/utils.js

```diff
@@ -879,21 +879,12 @@ export function updateSchemas(
     generateSchemaFromElementProps(elementArr),
   );
 
-  const existingUiSchema: {
-    [string]: any,
-    definitions?: { [string]: any },
-    ...
-  } = Object.entries(uischema)
-    .filter(([key, _value]) => {
-      return key in newSchema.properties || key.startsWith('ui:');
-    })
-    .reduce((accumulator, currentValue) => {
-      const [key, value] = currentValue;
-      return { ...accumulator, [key]: value };
-    }, {});
+  const definedUi = (uischema || {}).definitions
+    ? { definitions: uischema.definitions }
+    : {};
 
   const newUiSchema = Object.assign(
-    { ...existingUiSchema },
```

Comment:
The problem with this is that when going from Radio back to Short Answer, generateUiSchemaFromElementProps gives back a UI Schema with one fewer property, the intent being to remove that property, but appending to the existing UI schema undoes the removal.